gtk2_ardour/ardour.sh.in

```diff
@@ -61,8 +61,9 @@ export VAMP_PATH=@LIBDIR@/vamp${VAMP_PATH:+:$VAMP_PATH}
 
 if [ $# -gt 0 ] ; then
     case $1 in
-        -g|--gdb) GDB=gdb; shift ;;
+        -d|--ddd) DEBUGGER=ddd; shift ;;
```

Comment:
`-d` is an option for Ardour and should not be used by the wrapper script.

```
  -d, --disable-plugins       Disable all plugins (safe mode)
```

Reply:
Thank you. I will fix that. Was not aware.